IdSSLOpenSSLHeaders.pas

```diff
@@ -22939,9 +23196,8 @@ function Load: Boolean;
       LMinor := (LVersion and $0FF00000) shr 20;
       if (LMajor = 0) and (LMinor = 0) then begin // < 0.9.3
         LMajor := (LVersion and $F000) shr 12;
-        LMinor := (LVersion and $0F00) shr 8;

```

Comment:
I would comment out this line instead of removing it completely, in case we need the minor version in the future.